packages/kaizen-tg/_templates/drupal-9-theme/new/postcss.config.js

```diff
@@ -27,9 +27,30 @@ module.exports = () => ({
     }),
     postcssDiscardEmpty(),
     pxtorem({
+      minPixelValue: 3,
```

Comment:
i'm not sure all these changes are useful..
usually we use rems scaling. See example: on 1440px design you have border with 2px width. But what you will show on retina 5120px resolution? Your 2px will look tiny. That's why everything should be converted to rems

Reply:
For the CSS which is really required to be in pixel, you can use the following code:
```
.selector {
  /* stylelint-disable-next-line unit-case */
  width: 2PX;
}
```

In that case it will not be automatically converted to rems

Reply:
i would better set fixed base font size after 1920